src/locale/en.js

```diff
@@ -18,14 +18,37 @@ I18n.translations.en = {
     title: "Never stop learning",
     description: "Edubadges are digital certificates and show that the owner has certain skills or knowledge. Edubadges can be issued for both extracurriculair activities and accredited education.",
     student: {
-      title: "Receive and share badges",
-      button: "View your Backpack",
-      button_sub: "Log in with your eduID",
-      edu_id_info: "You will need an eduID to be able to view your Backpack. If you don’t have an eduID yet, you can easily <a target=\"_blank\" href=\"%{url}\">request one here.</a>",
+      title: "My Backpack",
+      subtitle: "receive and share badges",
+      action: "Open your Backpack",
+      button: "Log in with your eduID",
+      accountCreation: {
+        askAccountNo: "<strong>Don't</strong> have a Backpack yet?",
+        askAccountYes: "<strong>Already</strong> have a Backpack?",
+        startAccount: "Start here",
+        logInAccount: "Log in",
+        createAccountButton: "Create an eduID",
+        back: "Back",
+        title: "Create a Backpack",
```

Comment:
Maybe it makes sense to separate the locales for the new card from the student card, so that we'll have e.g. `student`, `createEduId`, `teacher`. What do you think?